### DISCUSSION TITLE
Why is sex-homosexuality a field on appdata.xml?

### DISCUSSION THREAD

**Discussion by ChildishGiant** [elementary/appcenter#1505] [Q&A] [answered]
Is it because of certain countries laws? I feel very odd needing to disclose it

**Comment by ChildishGiant** [comment]
I've found [this thread](https://github.com/hughsie/oars/issues/35) but I'd much rather not disclose either way.

**Comment by danirabbit** [comment, marked as answer]
This is something that is unfortunately a part of the current OARS spec due to some concerns about international law. elementary does not agree with the upstream maintainer here and we would like this to change.

I would encourage you not to include this field in your appdata. AppCenter will never do anything with this information.

**Comment by davidmhewitt** [comment]
I guess it's also worth clarifying that this XML field is designed to communicate whether the app contains or exposes any content that could be described as homosexual. 

I agree that it's an irrelevant field and should be omitted, but I'm just clarifying that it's not asking the author of the file to disclose their sexuality but to describe the content of the application. All of the OARS fields are about the content of the application.
